src/config/cohort.ts

```diff
@@ -0,0 +1,2 @@
+export const NEW_COHORT = "2023-2024";
+export const LEGACY_COHORT = "2020-2023";
```

Comment:
I'm guessing that legacy cohorts may grow in the future, will this better as an array ? Not sure myself...

Reply:
Lets keep it as a variable until we need more